impress-org/givewp-next-gen
# Feature: Auto-activate free license for updates

## Description



This PR auto-activates a free license for the Visual Donation Form Builder plugin for the purpose of automatic updates.

## Testing Instructions



- Activate the Visual Donation Form Builder plugin.
- The license should activate.
- If the license activation fails there should be a log entry.
